Review of MOM section in 25km config doc (#911)
COSIMA/MOM6-CICE6: documentation/docs/pages/configurations/MC_25km.md

### Frazil formation
Frazil ice typically consists of individual particles that appear like a coarse slush. Frazil ice can contribute to the creation of first-year sea ice, evolving to form a soupy ‘grease ice’ layer on the ocean surface, which may then develop into pancake ice. Frazil formation in the ocean is turned on (`FRAZIL = TRUE`), Frazil ice forms in the model when the in situ temperature drops below the local freezing point, taking into account the in situ salinity and pressure. This scheme works upwards through each water column, transferring heat downwards from the layer above as needed to prevent the in-situ temperature falling below the local freezing point in each layer in turn. If the top layer is below freezing, heat is extracted from the sea ice model, which grows frazil ice in response. In these configurations, we also have:
 
 1. `MASK_SRESTORE_UNDER_ICE = False`, i.e. do not disable SSS restoring under sea-ice based on a frazil criteria;
  1. `PRESSURE_DEPENDENT_FRAZIL = True` a pressure dependent freezing temperature is used when making frazil;
  1. `RECLAIM_FRAZIL = True` use any frazil heat deficit to cool any overlying layers down to the freezing point.

> 

Author: ```suggestion
Frazil ice typically consists of individual particles that appear like a coarse slush. Frazil ice can contribute to the creation of first-year sea ice, evolving to form a soupy ‘grease ice’ layer on the ocean surface, which may then develop into pancake ice. Frazil formation in the ocean is turned on (`FRAZIL = TRUE`). Frazil ice forms in the model when the in situ temperature drops below the local freezing point, taking into account the in situ salinity and pressure (`PRESSURE_DEPENDENT_FRAZIL = True`). The scheme works upwards through each water column, transferring heat downwards from the layer above as needed to prevent the in-situ temperature falling below the local freezing point in each layer in turn. If the top layer is below freezing, heat is extracted from the sea ice model, which grows frazil ice in response. In these configurations, we also have:
 
 1. `MASK_SRESTORE_UNDER_ICE = False`, i.e. do not disable SSS restoring under sea-ice based on a frazil criteria;
  1. `RECLAIM_FRAZIL = True` use any frazil heat deficit to cool any overlying layers down to the freezing point.

```

> More details are [here](https://mom6.readthedocs.io/en/main/api/generated/pages/Frazil_Ice.html.

Author: ```suggestion
More details are [here](https://mom6.readthedocs.io/en/main/api/generated/pages/Frazil_Ice.html).
```

The ePBL scheme parameters in the configuration are based on the [GFDL OM5 configuration](https://github.com/NOAA-GFDL/MOM6-examples/blob/3c1de3512e2200bfc10d9e5150715c9df76dbd30/ice_ocean_SIS2/Baltic_OM5_025/MOM_parameter_doc.all), including:

- Additional mixing due to Langmuir (wave-driven) turbulence (`EPBL_LANGMUIR_SCHEME = “ADDITIVE”`). Since we do not explicitly couple to a wave model in this configuration (`USE_WAVES = False`), the Langmuir effect is parameterised via a predetermined enhancement factor using MOM6 default values. The inclusion of wave effects is expected to reduce warm SST biases by enhancing mixing under strong winds, as found in studies of Langmuir turbulence (e.g., `USE_LA_LI2016 = True` from [@li2016langmuir]).
- Using the "OM4" scheme for calculating $m*$ (`EPBL_MSTAR_SCHEME = “OM4”`  `MSTAR_CAP = 1.25`, `MSTAR2_COEF1 = 0.29`, `MSTAR2_COEF2 = 0.152`)
- Iteratively solving for a self-consistent mixed layer depth rather than using a single-pass estimate (`USE_MLD_ITERATION = True`)
- Replacing shear-induced diffusivities with ePBL diffusivities when the latter is larger than the former (`EPBL_IS_ADDITIVE = False`)

#### Interior shear-driven mixing

> Shear-driven mixing is parameterised using the Jackson-Hallberg-Legg shear mixing scheme [@jackson2008parameterization] with the MOM6 default critical Richardson number (Ri) and shear mixing rate (`USE_JACKSON_PARAM = True`, `RINO_CRIT = 0.25` and `SHEARMIX_RATE = 0.089`). This scheme targets mixing in stratified shear zones, effectively adding interior vertical diffusivity when the local Ri is less than the critical value. The shear is computed at cell vertices to better capture shear between adjacent grid cells (`VERTEX_SHEAR = True`). The Jackson-Hallberg-Legg parameterisation is energetically constrained: it iteratively finds a diffusivity such that the energy extracted from the mean flow equals the energy used in mixing plus that lost to dissipation (`MAX_RINO_IT = 25`, inherited from the [GFDL OM5 configuration](https://github.com/NOAA-GFDL/MOM6-examples/blob/3c1de3512e2200bfc10d9e5150715c9df76dbd30/ice_ocean_SIS2/Baltic_OM5_025/MOM_parameter_doc.all#L2088)). See the [MOM6 documentation for further details](https://mom6.readthedocs.io/en/main/api/generated/pages/Internal_Vert_Mixing.html#shear-driven-mixing-in-jackson).

Author: ```suggestion
Shear-driven mixing is parameterised using the Jackson-Hallberg-Legg shear mixing scheme [@jackson2008parameterization] with the MOM6 default critical Richardson number (Ri) and shear mixing rate (`USE_JACKSON_PARAM = True`, `RINO_CRIT = 0.25` and `SHEARMIX_RATE = 0.089`). This scheme targets mixing in stratified shear zones, effectively adding interior vertical diffusivity when the local Ri is less than the critical value. The shear is computed at cell vertices to better capture shear between adjacent grid cells (`VERTEX_SHEAR = True`). The Jackson-Hallberg-Legg parameterisation is energetically constrained: it iteratively finds a diffusivity such that the energy extracted from the mean flow equals the energy used in mixing plus that lost to dissipation (`MAX_RINO_IT = 25`; inherited from the [GFDL OM5 configuration](https://github.com/NOAA-GFDL/MOM6-examples/blob/3c1de3512e2200bfc10d9e5150715c9df76dbd30/ice_ocean_SIS2/Baltic_OM5_025/MOM_parameter_doc.all#L2088)). See the [MOM6 documentation for further details](https://mom6.readthedocs.io/en/main/api/generated/pages/Internal_Vert_Mixing.html#shear-driven-mixing-in-jackson).
```

The Boussinesq approximation is used (`BOUSSINESQ = True`), meaning density variations only affect buoyancy, with other terms using a reference density `RHO_0 = 1035` $kg/m^3$.

### Thermodynamics and Equation of State (TEOS-10)
The configuration uses the "ROQUET_RHO" equation of state for seawater thermodynamic properties (`EQN_OF_STATE = "ROQUET_RHO"`). This scheme is based on TEOS-10, but uses a 75-term polynomial to compute in-situ density as a function of conservative temperature and absolute salinity, closely approximating the full TEOS-10 results [@roquet2015accurate]. The `_RHO` variant specifically fits density rather than specific volume and is well-suited for layered models. Prognostic temperature and salinity are conservative temperature and absolute salinity (`USE_CONTEMP_ABSSAL = True`), consistent with the equation of state.

> The freezing conservative temperature is calculated from absolute salinity and pressure using a 23-term polynomial fit refactored from the TEOS-10 package (`TFREEZE_FORM = "TEOS_POLY"`). More relevant discussion can be found in [COSIMA TWG 23-July-2025 meeting minutes](https://forum.access-hive.org.au/t/cosima-twg-meeting-minutes-2025/4067/17#:~:text=Freezing%20temperature%20consistency%20between%20mom6%20and%20cice). 

Author: ```suggestion
The freezing conservative temperature is calculated from absolute salinity and pressure using a 23-term polynomial fit refactored from the TEOS-10 package (`TFREEZE_FORM = "TEOS_POLY"`). More relevant discussion can be found in the [COSIMA TWG 23-July-2025 meeting minutes](https://forum.access-hive.org.au/t/cosima-twg-meeting-minutes-2025/4067/17#:~:text=Freezing%20temperature%20consistency%20between%20mom6%20and%20cice). 
```

### Frazil formation
Frazil formation in the ocean is turned on (`FRAZIL = TRUE`). This scheme works upwards through each water column, transferring heat downwards from the layer above as needed to prevent the in-situ temperature falling below the local freezing point in each layer in turn. If the top layer is below freezing, heat is extracted from the sea ice model, which grows frazil ice in response. More details are [here](https://mom6.readthedocs.io/en/main/api/generated/pages/Frazil_Ice.html).

In regions where there is no frazil formation, sea-ice melt/freeze potential is calculated over the smaller of the top 10m of the ocean and the boundary layer depth (`HFREEZE = 10.0`).

### Surface salinity restoring

> Sea surface salinity is restored toward a monthly climatological dataset calculated from the World Ocean Atlas 2023 available at [NOAA NCEI](https://www.ncei.noaa.gov/products/world-ocean-atlas) (`RESTORE_SALINITY = True`, `SALT_RESTORE_FILE = "salt_sfc_restore.nc"`). A piston velocity of 0.11 $m/day$ is applied to control the strength of the salinity relaxation (`FLUXCONST = 0.11`). The restoring is implemented as a virtual salt flux ( `SRESTORE_AS_SFLUX = True`). This approach conserves salt overall, balanced globally by subtracting the mean flux to avoid altering global salinity (`ADJUST_NET_SRESTORE_TO_ZERO = True`). No effective limit is applied to the salinity restoring applied (`MAX_DELTA_SRESTORE = 999`). More discussion can be found at [issues/350](https://github.com/ACCESS-NRI/access-om3-configs/issues/350), [issues/325](https://github.com/ACCESS-NRI/access-om3-configs/issues/325), [issues/257](https://github.com/ACCESS-NRI/access-om3-configs/issues/257).

Author: ```suggestion
Sea surface salinity is restored toward a monthly climatological dataset calculated from the World Ocean Atlas 2023 available at [NOAA NCEI](https://www.ncei.noaa.gov/products/world-ocean-atlas) (`RESTORE_SALINITY = True`, `SALT_RESTORE_FILE = "salt_sfc_restore.nc"`). A piston velocity of 0.11 $m/day$ is applied to control the strength of the salinity relaxation (`FLUXCONST = 0.11`). The restoring is implemented as a virtual salt flux (`SRESTORE_AS_SFLUX = True`). This approach conserves salt overall, balanced globally by subtracting the mean flux to avoid altering global salinity (`ADJUST_NET_SRESTORE_TO_ZERO = True`). No effective limit is applied to the salinity restoring flux (`MAX_DELTA_SRESTORE = 999`). More discussion can be found in [issues/350](https://github.com/ACCESS-NRI/access-om3-configs/issues/350), [issues/325](https://github.com/ACCESS-NRI/access-om3-configs/issues/325), [issues/257](https://github.com/ACCESS-NRI/access-om3-configs/issues/257).
```

Salinity is limited to be positive to prevent the sea-ice model from asking for more salt than is available and driving the salinity negative (`BOUND_SALINITY = True`, `MIN_SALINITY = 0.0`).

### Diagnostics
Three-dimensional ocean diagnostics are output on either $z*$- or density-coordinates, depending on the diagnostic, rather than on the model's native coordinate. Specifically, `NUM_DIAG_COORDS = 2` with `DIAG_COORDS = "z Z ZSTAR", "rho2 RHO2 RHO"` and the vertical coordinate levels for each are defined by `DIAG_COORD_DEF_Z = "FILE:ocean_vgrid.nc,interfaces=zeta"` and `DIAG_COORD_DEF_RHO2 = "RFNC1:76,999.5,1020.,1034.1,3.1,1041.,0.002"` (relevant info can be found at [PR/622](https://github.com/ACCESS-NRI/access-om3-configs/pull/622)).

An ideal age tracer is configured (`USE_IDEAL_AGE_TRACER = True`). This tracer ages at a rate of 1/year once it is isolated from the surface and is useful for  understanding water mass ventilation and residence times.

### Vertical mixing parameterisations
#### Energetic planetary boundary layer (ePBL)

> The configuration handles the vertical mixing in the ocean surface boundary layer with the ePBL scheme rather the the traditional KPP. The ePBL scheme is an energy-based 1D turbulence closure approach that integrates a boundary layer energy budget to determine mixing coefficients. It was developed by [@reichl2018simplified] to improve upon KPP for climate simulations by including the effect of turbulent kinetic energy input and wind-driven mixing in a more physically constrained way. Relevant discussion can be found at [issues/465](https://github.com/ACCESS-NRI/access-om3-configs/issues/465), [issues/426](https://github.com/ACCESS-NRI/access-om3-configs/issues/426), [issues/373](https://github.com/ACCESS-NRI/access-om3-configs/issues/373).

Author: ```suggestion
The configuration handles the vertical mixing in the ocean surface boundary layer with the ePBL scheme rather the the traditional KPP. The ePBL scheme is an energy-based 1D turbulence closure approach that integrates a boundary layer energy budget to determine mixing coefficients. It was developed by [@reichl2018simplified] to improve upon KPP for climate simulations by including the effect of turbulent kinetic energy input and wind-driven mixing in a more physically constrained way. Relevant discussion can be found in [issues/465](https://github.com/ACCESS-NRI/access-om3-configs/issues/465), [issues/426](https://github.com/ACCESS-NRI/access-om3-configs/issues/426), [issues/373](https://github.com/ACCESS-NRI/access-om3-configs/issues/373).
```

## MOM6 parameter choices

Code-formatted text in the following sections gives the parameter values set in the `MOM_input` configuration file.

### Grid resolution

> The configuration uses a zonally-periodic tripolar grid to avoid a singularity at the North Pole (`REENTRANT_X = True`, `TRIPOLAR_N = True`). The horizontal grid has 1440 x 1152 tracer points providing eddy-permitting resolution similar to ACCESS-OM2-025 and GFDL's OM4/OM5 global models (1440 x 1080). See [Grids](../../inputs/Grids/) for more information. 

Reviewer: On this bit
> GFDL's OM4/OM5 global models (1440 x 1080)

Can we have a link for that?

Reviewer 2: Here's the link https://github.com/NOAA-GFDL/MOM6-examples/blob/f33849827fba879cf0a2ef39aba11822daef93f1/ice_ocean_SIS2/OM4_025/MOM_parameter_doc.all#L219-L224

> There are 75 vertical layers (`NK = 75`). MOM6 supports an arbitrary Lagrangian Euler (ALE) vertical coordinate scheme [@griffies2020primer] that allows for completely general vertical coordinates. A number of coordinates are supported "out-of-the-box", including isopycnal, geopotential, terrain-following, HYCOM1 etc. This configuration uses ALE with a stretched geopotential (z-star) vertical coordinate (`USE_REGRIDDING = True`, `REGRIDDING_COORDINATE_MODE = "ZSTAR"`). The layer spacing specified via an input file (`ALE_COORDINATE_CONFIG = "FILE:ocean_vgrid.nc,interfaces=zeta"`). The deepest ocean depth is set to `MAXIMUM_DEPTH = 6000.0`.

Reviewer: Thanks @ezhilsabareesh8. Can I suggest you add that detail?

I note that [currently we have](https://access-om3-configs.access-hive.org.au/inputs/Grids/#vertical-grid):
> Vertical grid
In the ocean model, we use a [75 level vertical grid](https://github.com/COSIMA/om3-scripts/blob/main/grid_generation/generate_vertical_grid.py) unchanged from many OM2 configurations, following Stewart et al.(2017)(Stewart et al., 2017)[2](https://access-om3-configs.access-hive.org.au/inputs/Grids/#fn:StewartHoggGriffiesHeerdegenWardSpenceEngland2017a).

I think putting it here^ makes more sense? Then just adding a link from here. If you do want to add it [here](https://access-om3-configs.access-hive.org.au/inputs/Grids/#vertical-grid), please do it on this PR:
https://github.com/ACCESS-NRI/access-om3-configs/pull/912/
(Just add your commit to the branch.)

Is it possible to even list what the layers are (as in what they start at)?

@dougiesquire has just been looking at how this varies with z* -- a comment in that sense might be nice too but not necessary.

> There are 75 vertical layers (`NK = 75`). MOM6 supports an arbitrary Lagrangian Euler (ALE) vertical coordinate scheme [@griffies2020primer] that allows for completely general vertical coordinates. A number of coordinates are supported "out-of-the-box", including isopycnal, geopotential, terrain-following, HYCOM1 etc. This configuration uses ALE with a stretched geopotential (z-star) vertical coordinate (`USE_REGRIDDING = True`, `REGRIDDING_COORDINATE_MODE = "ZSTAR"`). The layer spacing specified via an input file (`ALE_COORDINATE_CONFIG = "FILE:ocean_vgrid.nc,interfaces=zeta"`). The deepest ocean depth is set to `MAXIMUM_DEPTH = 6000.0`.

Reviewer: Thanks @ezhilsabareesh8 can you please add a reference to that here too? "Further details of the vertical grid can be found..." or similar.

> There are 75 vertical layers (`NK = 75`). MOM6 supports an arbitrary Lagrangian Euler (ALE) vertical coordinate scheme [@griffies2020primer] that allows for completely general vertical coordinates. A number of coordinates are supported "out-of-the-box", including isopycnal, geopotential, terrain-following, HYCOM1 etc. This configuration uses ALE with a stretched geopotential (z-star) vertical coordinate (`USE_REGRIDDING = True`, `REGRIDDING_COORDINATE_MODE = "ZSTAR"`). The layer spacing specified via an input file (`ALE_COORDINATE_CONFIG = "FILE:ocean_vgrid.nc,interfaces=zeta"`). The deepest ocean depth is set to `MAXIMUM_DEPTH = 6000.0`. Further details of the vertical grid can be found [here](https://access-om3-configs.access-hive.org.au/inputs/Grids/#vertical-grid).

Reviewer: ```suggestion
There are 75 vertical layers (`NK = 75`). MOM6 supports an arbitrary Lagrangian Euler (ALE) vertical coordinate scheme [@griffies2020primer] that allows for completely general vertical coordinates. A number of coordinates are supported "out-of-the-box", including isopycnal, geopotential, terrain-following, HYCOM1 etc. This configuration uses ALE with a stretched geopotential (z-star) vertical coordinate (`USE_REGRIDDING = True`, `REGRIDDING_COORDINATE_MODE = "ZSTAR"`). The layer spacing specified via an input file (`ALE_COORDINATE_CONFIG = "FILE:ocean_vgrid.nc,interfaces=zeta"`). The deepest ocean depth is set to `MAXIMUM_DEPTH = 6000.0`. Further details of the vertical grid can be found [here](../../inputs/Grids/#vertical-grid).
```

The Boussinesq approximation is used (`BOUSSINESQ = True`), meaning density variations only affect buoyancy, with other terms using a reference density `RHO_0 = 1035` $kg/m^3$.

### Thermodynamics and Equation of State (TEOS-10)

> The configuration uses the "ROQUET_RHO" equation of state for seawater thermodynamic properties (`EQN_OF_STATE = "ROQUET_RHO"`). This scheme is based on TEOS-10, but uses a 75-term polynomial to compute in-situ density as a function of conservative temperature and absolute salinity, closely approximating the full TEOS-10 results [@roquet2015accurate]. The `_RHO` variant specifically fits density rather than specific volume and is well-suited for layered models. Prognostic temperature and salinity are conservative temperature and absolute salinity (`USE_CONTEMP_ABSSAL = True`), consistent with the equation of state.

Reviewer: ```suggestion
The configuration uses the "ROQUET_RHO" equation of state for seawater thermodynamic properties (`EQN_OF_STATE = "ROQUET_RHO"`). This scheme is based on TEOS-10 using a 75-term polynomial to compute in-situ density as a function of conservative temperature and absolute salinity, closely approximating the full TEOS-10 results [@roquet2015accurate]. The `_RHO` variant specifically fits density rather than specific volume and is well-suited for layered models. Prognostic temperature and salinity are conservative temperature and absolute salinity (`USE_CONTEMP_ABSSAL = True`), consistent with the equation of state.
```

Reviewer: Currently we have:
https://access-om3-configs.access-hive.org.au/pr-preview-912/inputs/Initial-conditions-generation/

> The workflow produces Conservative Temperature (CT) and Absolute Salinity (SA), the prognostic variables required by the TEOS-10 equation of state (EOS) for use in ACCESS-OM3.

I think it would be worth mentioning or linking to that here.

For consistency, would also be re-wording

Sea surface salinity is restored toward a monthly climatological dataset calculated from the World Ocean Atlas 2023 available at [NOAA NCEI](https://www.ncei.noaa.gov/products/world-ocean-atlas) (`RESTORE_SALINITY = True`, `SALT_RESTORE_FILE = "salt_sfc_restore.nc"`). A piston velocity of 0.11 $m/day$ is applied to control the strength of the salinity relaxation (`FLUXCONST = 0.11`). The restoring is implemented as a virtual salt flux (`SRESTORE_AS_SFLUX = True`). This approach conserves salt overall, balanced globally by subtracting the mean flux to avoid altering global salinity (`ADJUST_NET_SRESTORE_TO_ZERO = True`). No effective limit is applied to the salinity restoring flux (`MAX_DELTA_SRESTORE = 999`). More discussion can be found in [issues/350](https://github.com/ACCESS-NRI/access-om3-configs/issues/350), [issues/325](https://github.com/ACCESS-NRI/access-om3-configs/issues/325), [issues/257](https://github.com/ACCESS-NRI/access-om3-configs/issues/257).

Salinity is limited to be positive to prevent the sea-ice model from asking for more salt than is available and driving the salinity negative (`BOUND_SALINITY = True`, `MIN_SALINITY = 0.0`).

### Diagnostics

> Three-dimensional ocean diagnostics are output on either $z*$- or density-coordinates, depending on the diagnostic, rather than on the model's native coordinate. Specifically, `NUM_DIAG_COORDS = 2` with `DIAG_COORDS = "z Z ZSTAR", "rho2 RHO2 RHO"` and the vertical coordinate levels for each are defined by `DIAG_COORD_DEF_Z = "FILE:ocean_vgrid.nc,interfaces=zeta"` and `DIAG_COORD_DEF_RHO2 = "RFNC1:76,999.5,1020.,1034.1,3.1,1041.,0.002"` (relevant info can be found at [PR/622](https://github.com/ACCESS-NRI/access-om3-configs/pull/622)).

Reviewer: Would it be possible to give a link(s) or some information as to which variables are output on which coordinates?

Author: We can add a link to the section on [ocean diagnostics](https://github.com/ACCESS-NRI/access-om3-configs/pull/1020) once it is merged

### Vertical mixing parameterisations
#### Energetic planetary boundary layer (ePBL)
The configuration handles the vertical mixing in the ocean surface boundary layer with the ePBL scheme rather the the traditional KPP. The ePBL scheme is an energy-based 1D turbulence closure approach that integrates a boundary layer energy budget to determine mixing coefficients. It was developed by [@reichl2018simplified] to improve upon KPP for climate simulations by including the effect of turbulent kinetic energy input and wind-driven mixing in a more physically constrained way. Relevant discussion can be found in [issues/465](https://github.com/ACCESS-NRI/access-om3-configs/issues/465), [issues/426](https://github.com/ACCESS-NRI/access-om3-configs/issues/426), [issues/373](https://github.com/ACCESS-NRI/access-om3-configs/issues/373).

The ePBL scheme parameters in the configuration are based on the [GFDL OM5 configuration](https://github.com/NOAA-GFDL/MOM6-examples/blob/3c1de3512e2200bfc10d9e5150715c9df76dbd30/ice_ocean_SIS2/Baltic_OM5_025/MOM_parameter_doc.all), including:

> - Additional mixing due to Langmuir (wave-driven) turbulence (`EPBL_LANGMUIR_SCHEME = “ADDITIVE”`). Since we do not explicitly couple to a wave model in this configuration (`USE_WAVES = False`), the Langmuir effect is parameterised via a predetermined enhancement factor using MOM6 default values. The inclusion of wave effects is expected to reduce warm SST biases by enhancing mixing under strong winds, as found in studies of Langmuir turbulence (e.g., `USE_LA_LI2016 = True` from [@li2016langmuir]).

Reviewer: > The inclusion of wave effects is expected to reduce warm SST biases by enhancing mixing under strong winds

is that what we've found in `mcw` @ezhilsabareesh8 @NoahDay ?

> - Additional mixing due to Langmuir (wave-driven) turbulence (`EPBL_LANGMUIR_SCHEME = “ADDITIVE”`). Since we do not explicitly couple to a wave model in this configuration (`USE_WAVES = False`), the Langmuir effect is parameterised via a predetermined enhancement factor using MOM6 default values. The inclusion of wave effects is expected to reduce warm SST biases by enhancing mixing under strong winds, as found in studies of Langmuir turbulence (e.g., `USE_LA_LI2016 = True` from [@li2016langmuir]).

Reviewer 3: When WW3 is fully coupled to ePBL vertical mixing, we find that the resulting mixed-layer depths are unrealistic; therefore, we disable wave-induced mixing in ePBL (i.e., no wave to ocean coupling) to obtain reasonable mixed-layer depths.

An additional Rayleigh drag is applied to layers within the bottom boundary layer to account for curvature of the bottom (`CHANNEL_DRAG = True`, `SMAG_CONST_CHANNEL = 0.15`). More details can be found [here](https://mom6.readthedocs.io/en/main/api/generated/pages/Vertical_Viscosity.html?highlight=channel%20drag#channel-drag).

### Horizontal viscosity and subgrid momentum mixing
A hybrid Laplacian-biharmonic viscosity scheme is used to parameterise unresolved horizontal turbulent mixing of momentum (`LAPLACIAN = True`, `BIHARMONIC = True`). The scheme helps remove small-scale kinetic energy, while preserving large-scale eddy structures, targetting the smaller scales more selectively than just using a Laplacian scheme. See the [MOM6 documentation](https://mom6.readthedocs.io/en/main/api/generated/modules/mom_hor_visc.html#namespacemom-hor-visc-1section-horizontal-viscosity:~:text=Laplacian%20viscosity%20coefficient) for details of how the horizontal viscosity is calculated. The biharmonic viscosity includes:

- no constant background viscosity (`AH = 0.0`)

> - a grid-dependent background viscosity (`AH_TIME_SCALE = 0.0`, `AH_TIME_SCALE = 0.0`)

Author: ```suggestion
- a grid-dependent background viscosity (`AH_VEL_SCALE = 0.01`, `AH_TIME_SCALE = 0.0`)
```

- no constant background viscosity (`AH = 0.0`)
- a grid-dependent background viscosity (`AH_VEL_SCALE = 0.01`, `AH_TIME_SCALE = 0.0`)
- a dynamic Smagorinsky nonlinear eddy viscosity (`SMAGORINSKY_AH = True`, `SMAG_BI_CONST = 0.06`, `LEITH_AH = False`)

The Lapacian viscosity includes:

- no constant background viscosity (`KH = 0.0`)
- a grid-dependent background viscosity (`KH_VEL_SCALE = 0.01`)
- a latitudinally-dependent background viscosity (`KH_SIN_LAT = 2000.0`, `KH_PWR_OF_SINE = 4.0`)
- no file-base background viscosity (`USE_KH_BG_2D = False`)
- no dynamic viscosity component (`SMAGORINSKY_KH = False`, `LEITH_KH = False`)
- reduction scaling in well-resolved regions (`RESOLN_SCALED_KH = True`) 
- the two coefficient anisotropic viscosity scheme proposed by [@smith2003anisotropic] is not used (`ANISOTROPIC_VISCOSITY = False`)
The Laplacian and biharmonic coefficients are both limited locally to guarantee stability (`BOUND_KH = True`, `BETTER_BOUND_KH = True`, `BOUND_AH = True`, `BETTER_BOUND_AH = True`).

### Isopycnal mixing

> Baroclinic instability converts available potential energy (APE) stored in sloping isopycnals into eddy kinetic energy (EKE). In an eddy-permitting 25km configuration, this conversion is only partly resolved hence the model does not fully capture the eddy processes that naturally flatten isopycnals and release APE. As a result, isopycnal slopes remain steeper than they should be unless the unresolved eddy effects are parameterised. More details can be found in [@mitgcm_gmredi].

Reviewer 2: Sorry for the confusion. I meant details of the physical basis and formulation of GM. 

```suggestion
Baroclinic instability converts available potential energy (APE) stored in sloping isopycnals into eddy kinetic energy (EKE). In an eddy-permitting 25km configuration, this conversion is only partly resolved hence the model does not fully capture the eddy processes that naturally flatten isopycnals and release APE. As a result, isopycnal slopes remain steeper than they should be unless the unresolved eddy effects are parameterised. More details on the physical basis and mathematical formulation of the GM parameterisation can be found in [@mitgcm_gmredi].
```

Mesoscale eddies also induce irreversible mixing of tracers but primarily along neutral density surfaces rather than vertically. This diabatic component needs to be represented through an isopycnal diffusion parameterisation that diffuses tracers along neutral surfaces while avoiding spurious diapycnal mixing.

> To represent both the flattening of isopycnals and the along-isopycnal tracer mixing, the configuration applies a hybrid mesoscale parameterisation that combines neutral diffusion [@redi1982oceanic] with the Gent-McWilliams (`GM`) eddy-induced advection scheme [@gent1990isopycnal]. Both require an eddy diffusivity,

Reviewer 2: ```suggestion
To represent both the quasi-adiabatic flattening of isopycnals and along-isopycnal tracer mixing, the configuration applies a hybrid mesoscale parameterisation that combines neutral diffusion [@redi1982oceanic] with the Gent-McWilliams (`GM`) eddy-induced advection scheme [@gent1990isopycnal]. Both schemes require an eddy diffusivity,
```

- thickness diffusivity ($KHTH$ in MOM6)
- isopycnal tracer diffusivity ($KHTR$ in MOM6)

> In MOM5, these coefficients were prescribed constants or latitude-dependent maps, but these choices are ad-hoc and not dynamically constrained. Hence MOM6 also offers the Mesoscale Eddy Kinetic Energy (MEKE) scheme [@mom6_mom_meke] which provides a flow-dependent, scale-aware GM diffusivity. MEKE prognostically computes an eddy kinetic energy field $E(x,y,z,t)$ from which it derives an eddy velocity scale,

Reviewer 2: ```suggestion
In MOM5, these coefficients were prescribed constants or latitude-dependent maps, but these choices are ad-hoc and not dynamically constrained. Hence MOM6 also offers the Mesoscale Eddy Kinetic Energy (`MEKE`) scheme [@mom6_mom_meke] which provides a flow-dependent, scale-aware `GM` diffusivity. `MEKE` prognostically computes an eddy kinetic energy field $E(x,y,z,t)$ from which it derives an eddy velocity scale,
```

$$
U_{eddy}=\sqrt{2E}
$$

and an eddy length scale ($L$) based on a configurable combination of multiple length scales. The GM thickness diffusivity is then computed as,

$$

> KHTH = CU_{eddy}L

Author: ```suggestion
KHTH = CU_{e}L
```

$$,

> Hence when EKE is large, GM diffusivity increases with stronger flattening of isopycnals and similarly when EKE is small, GM diffusivity decreases and let the resolved eddies do the work. Hence the GM flattening of isopycnals becomes energetically consistent and scale-aware, adapting automatically to the local eddy field.

Reviewer 2: ```suggestion
Hence when EKE is large, `GM` diffusivity increases with stronger flattening of isopycnals and similarly when EKE is small, `GM` diffusivity decreases and let the resolved eddies do the work. Hence the `GM` flattening of isopycnals becomes energetically consistent and scale-aware, adapting automatically to the local eddy field.
```

> KHTH = CU_{eddy}L

Reviewer 2: Done in [fffa729](https://github.com/ACCESS-NRI/access-om3-configs/pull/911/commits/fffa72998ad043d1bcb8db5b473521ddd3a5d4a8)

#### Isopycnal thickness diffusion (Gent McWilliams bolus transport)
In MOM6, `GM` is implemented in a thickness diffusion form. MOM6 adds a diffusive flux of layer thickness to the thickness (mass continuity) equation. This height diffusion formulation is mathematically equivalent to the original scheme, which was expressed as an eddy-induced (bolus) velocity that advects tracers and layer thickness [@adcroft2019gfdl]. `GM` is turned on via `THICKNESSDIFFUSE = True`. 

> This configuration uses `MEKE` (`USE_MEKE = True`) to provide the `GM` diffusivity. Because we do not supply an external EKE field (`EKE_SOURCE = "prog"`), EKE is generated internally through instability growth. `MEKE_BGSRC = 1.0E-13` prevents `EKE` from decaying to zero in very quiet regions. It serves as a floor to aid numerical stability and is analogous to a background diffusivity but in energy form. `MEKE_GMCOEFF = 1.0` means the scheme converts eddy potential energy to eddy kinetic energy with 100% efficiency for the `GM` effect. `MEKE_KHTR_FAC = 0.5` and `MEKE_KHTH_FAC = 0.5` map some of the eddy energy to tracer diffusivity and thickness diffusivity, respectively. In practice, `MEKE` supplies the dynamically varying `GM` coefficient that flattens isopycnals and removes APE in a physically informed way. We use `KHTH_USE_FGNV_STREAMFUNCTION = True`, which solves a 1-D boundary-value problem to ensure the `GM` streamfunction is smooth in the vertical and vanishes at the surface and bottom [@ferrari2010boundary]. `FGNV_FILTER_SCALE = 0.1` applies light spatial filtering to reduce noise in the diagnosed streamfunction. We set `RES_SCALE_MEKE_VISC = False`, meaning viscosity is not explicitly scaled by `MEKE`.

Reviewer 2: No action required

> With `MEKE`, MOM6 becomes explicitly resolution-aware: as horizontal resolution increases and more eddy processes are partially resolved, the diagnosed GM coefficient naturally decreases; conversely, GM strengthens where eddies remain under-resolved (e.g. high latitudes), removing the need for ad-hoc spatial maps of GM coefficients.

Reviewer 2: ```suggestion
With `MEKE`, MOM6 becomes explicitly resolution-aware: as horizontal resolution increases and more eddy processes are partially resolved, the diagnosed `GM` coefficient naturally decreases; conversely, `GM` strengthens where eddies remain under-resolved (e.g. high latitudes), removing the need for ad-hoc spatial maps of `GM` coefficients.
```

### Frazil formation
Frazil ice typically consists of individual particles that appear like a coarse slush. Frazil ice can contribute to the creation of first-year sea ice, evolving to form a soupy ‘grease ice’ layer on the ocean surface, which may then develop into pancake ice. Frazil formation in the ocean is turned on (`FRAZIL = TRUE`, `ENABLE_THERMODYNAMICS = True`), Frazil ice forms in the model when the in situ temperature drops below the local freezing point, taking into account the in situ salinity and pressure. This scheme works upwards through each water column, transferring heat downwards from the layer above as needed to prevent the in-situ temperature falling below the local freezing point in each layer in turn. If the top layer is below freezing, heat is extracted from the sea ice model, which grows frazil ice in response. In these configurations, we also have:
 

>  1. `MASK_SRESTORE_UNDER_ICE = False`, i.e. disables SSS restoring under sea-ice based on a frazil criteria;

Author: ```suggestion
 1. `MASK_SRESTORE_UNDER_ICE = False`, i.e. do not disable SSS restoring under sea-ice based on a frazil criteria;
```

The ePBL scheme parameters in the configuration are based on the [GFDL OM5 configuration](https://github.com/NOAA-GFDL/MOM6-examples/blob/3c1de3512e2200bfc10d9e5150715c9df76dbd30/ice_ocean_SIS2/Baltic_OM5_025/MOM_parameter_doc.all), including:

- Additional mixing due to Langmuir (wave-driven) turbulence (`EPBL_LANGMUIR_SCHEME = “ADDITIVE”`). Since we do not explicitly couple to a wave model in this configuration (`USE_WAVES = False`), the Langmuir effect is parameterised via a predetermined enhancement factor using MOM6 default values. The inclusion of wave effects is expected to reduce warm SST biases by enhancing mixing under strong winds, as found in studies of Langmuir turbulence (e.g., `USE_LA_LI2016 = True` from [@li2016langmuir]).
- Using the "OM4" scheme for calculating $m*$ (`EPBL_MSTAR_SCHEME = “OM4”`  `MSTAR_CAP = 1.25`, `MSTAR2_COEF1 = 0.29`, `MSTAR2_COEF2 = 0.152`)
- Iteratively solving for a self-consistent mixed layer depth rather than using a single-pass estimate (`USE_MLD_ITERATION = True`)
- Replacing shear-induced diffusivities with ePBL diffusivities when the latter is larger than the former (`EPBL_IS_ADDITIVE = False`)

#### Interior shear-driven mixing

> Shear-driven mixing is parameterised using the Jackson-Hallberg-Legg shear mixing scheme [@jackson2008parameterization] with the MOM6 default critical Richardson number (Ri; `RINO_CRIT = 0.25`) and shear mixing rate (`USE_JACKSON_PARAM = True` and `SHEARMIX_RATE = 0.089`). This scheme targets mixing in stratified shear zones, effectively adding interior vertical diffusivity when the local Ri is less than the critical value. The shear is computed at cell vertices to better capture shear between adjacent grid cells (`VERTEX_SHEAR = True`). The Jackson-Hallberg-Legg parameterisation is energetically constrained: it iteratively finds a diffusivity such that the energy extracted from the mean flow equals the energy used in mixing plus that lost to dissipation (`MAX_RINO_IT = 25`, inherited from the [GFDL OM5 configuration](https://github.com/NOAA-GFDL/MOM6-examples/blob/3c1de3512e2200bfc10d9e5150715c9df76dbd30/ice_ocean_SIS2/Baltic_OM5_025/MOM_parameter_doc.all#L2088)). See the [MOM6 documentation for further details](https://mom6.readthedocs.io/en/main/api/generated/pages/Internal_Vert_Mixing.html#shear-driven-mixing-in-jackson).

Author: ```suggestion
Shear-driven mixing is parameterised using the Jackson-Hallberg-Legg shear mixing scheme [@jackson2008parameterization] with the MOM6 default critical Richardson number (Ri) and shear mixing rate (`USE_JACKSON_PARAM = True`, `RINO_CRIT = 0.25` and `SHEARMIX_RATE = 0.089`). This scheme targets mixing in stratified shear zones, effectively adding interior vertical diffusivity when the local Ri is less than the critical value. The shear is computed at cell vertices to better capture shear between adjacent grid cells (`VERTEX_SHEAR = True`). The Jackson-Hallberg-Legg parameterisation is energetically constrained: it iteratively finds a diffusivity such that the energy extracted from the mean flow equals the energy used in mixing plus that lost to dissipation (`MAX_RINO_IT = 25`, inherited from the [GFDL OM5 configuration](https://github.com/NOAA-GFDL/MOM6-examples/blob/3c1de3512e2200bfc10d9e5150715c9df76dbd30/ice_ocean_SIS2/Baltic_OM5_025/MOM_parameter_doc.all#L2088)). See the [MOM6 documentation for further details](https://mom6.readthedocs.io/en/main/api/generated/pages/Internal_Vert_Mixing.html#shear-driven-mixing-in-jackson).
```
